src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/overlays/mediacropdetails.html

```diff
@@ -26,15 +26,15 @@ <h5>
 
             <div class="umb-control-group" ng-if="model.target">
 
-                <div ng-if="model.disableFocalPoint && model.target.thumbnail">
+                <div ng-if="vm.cropSet() === false">
                     <h5>
                         <localize key="general_preview">Preview</localize>
                     </h5>
 
                     <img ng-src="{{model.target.thumbnail}}" alt="{{model.target.name}}" />
                 </div>
 
-                <div ng-if="!model.disableFocalPoint">
+                <div ng-if="vm.cropSet()">
```

Comment:
@nul800sebastiaan couldn't this just run the function in the controller?

```
vm.hasCrops = cropSet() === true;
```

Then use `vm.hasCrops` in view and remove this in controller `vm.cropSet = cropSet;`

With the current logic it needs to evaluate `cropSet()` twice.

Reply:
Thanks @bjarnef - I've fixed this up now. https://github.com/umbraco/Umbraco-CMS/commit/e1ef5f4e6b6b28715d965a9a3d2dac951f251de6